include/ebpf_structs.h

```diff
@@ -218,6 +218,16 @@ enum bpf_prog_type
      */
     BPF_PROG_TYPE_SOCK_OPS,
 
+    /** @brief Program type for handling incoming packets as early as possible.
+     *
+     * **eBPF program prototype:** \ref xdp_hook_t
+     *
+     * **Attach type(s):** \ref BPF_XDP_TEST
+     *
+     * **Helpers available:** all helpers defined in bpf_helpers.h
+     */
+    BPF_PROG_TYPE_XDP_TEST,
```

Comment:
Below has BPF_PROG_TYPE_SAMPLE = 999, so maybe this should be at the end too?  like
BPF_PROG_TYPE_SAMPLE_XDP = 998 ?